core/services/chainlink/application.go

```diff
@@ -222,21 +221,10 @@ func NewApplication(opts ApplicationOpts) (Application, error) {
 
 	telemetryIngressClient := synchronization.TelemetryIngressClient(&synchronization.NoopTelemetryIngressClient{})
 	telemetryIngressBatchClient := synchronization.TelemetryIngressBatchClient(&synchronization.NoopTelemetryIngressBatchClient{})
-	explorerClient := synchronization.ExplorerClient(&synchronization.NoopExplorerClient{})
 	monitoringEndpointGen := telemetry.MonitoringEndpointGenerator(&telemetry.NoopAgent{})
 
-	if cfg.Explorer().URL() != nil && cfg.TelemetryIngress().URL() != nil {
-		globalLogger.Warn("Both ExplorerUrl and TelemetryIngress.Url are set, defaulting to Explorer")
-	}
-
-	if cfg.Explorer().URL() != nil {
-		explorerClient = synchronization.NewExplorerClient(cfg.Explorer().URL(), cfg.Explorer().AccessKey(), cfg.Explorer().Secret(), globalLogger)
-		monitoringEndpointGen = telemetry.NewExplorerAgent(explorerClient)
-	}
-
 	ticfg := cfg.TelemetryIngress()
-	// Use Explorer over TelemetryIngress if both URLs are set
-	if cfg.Explorer().URL() == nil && ticfg.URL() != nil {
+	if ticfg.URL() != nil {
 		if ticfg.UseBatchSend() {
 			telemetryIngressBatchClient = synchronization.NewTelemetryIngressBatchClient(ticfg.URL(),
```

Comment:
unrelated to the scope of this PR, but this is a pretty heavy API w/ a large number of the parameters coming from `cfg.TelemetryIngress()` - what do you think of plumbing `ticfg` as one parameter through instead?

Reply:
sounds good to me, that would definitely clean things up a bit 🙂